packages/core/src/components/account/profile/profile.module.scss

```diff
@@ -0,0 +1,89 @@
+@layer components {
+  .profileSection {
```

Comment:
What about just `section` to keep the pattern?  Also, is it worth to create any [local tokens](https://developers.vtex.com/docs/guides/faststore/using-themes-components)?

Reply:
What is the section pattern? Just call this className `.section`? Idk about the local tokens - what is the criteria for the their creation?

Reply:
Yes, just call it `section` as the other sections. Sometimes we also use `.page` in specific scenarios. 

We can go ahead with just using the globals, I'd rather use locals because the other sections used to have local tokens for sections, and then users can change the local token when they decide to change the style.